docs/quickstart.html

```diff
@@ -32,8 +32,8 @@ <h4 class="anchor-heading">
             the latest Kafka release and extract it:
         </p>
 
-        <pre><code class="language-bash">$ tar -xzf kafka_{{scalaVersion}}-{{fullDotVersion}}.tgz
-$ cd kafka_{{scalaVersion}}-{{fullDotVersion}}</code></pre>
+        <pre><code class="language-bash">tar -xzf kafka_{{scalaVersion}}-{{fullDotVersion}}.tgz
```

Comment:
Why are we removing the `$` prompts? We use them everywhere in the documentation. It's useful to distinguish the commands from their outputs.

Reply:
@jsancio : I agree, let's not make this change. (Or if you really want to do it, let's create a separate PR where we can discuss further, and do it for all docs.)